src/Reflection/Php/PhpFunctionFromParserNodeReflection.php

```diff
@@ -137,7 +137,7 @@ private function getParameters(): array
 				$parameter->var->name,
 				$isOptional,
 				$this->realParameterTypes[$parameter->var->name],
-				isset($this->phpDocParameterTypes[$parameter->var->name]) ? $this->phpDocParameterTypes[$parameter->var->name] : null,
+				$this->phpDocParameterTypes[$parameter->var->name] ?? null,
```

Comment:
@kukulich Here are some improvements we can make to `RequireNullCoalesceOperator` :innocent: